cheatle/__main__.py

```diff
@@ -1,2 +1,24 @@
+from curses.ascii import isalpha
+from email.policy import default
+import click
+
+#function that takes in the known/green values
+#would like to set a default value to "_____" but when I do that within the click option "Green Letters:  [_____]:" is displayed as the prompt
+@click.command()
+@click.option('--known', prompt = "Green Letters", help='Letters that are in the correct position. Use a "_" for letters that are not in the right position')
+def known_word(known):
+    #check to make sure user only used letters and "_" for known
+    for i in known:
```

Comment:
The validation should not be in the command method implementation. Once we have lots of parameters this will get very long and hard to maintain. I suggest using the callbacks for validation like in this [documentation]( https://click.palletsprojects.com/en/8.1.x/options/#callbacks-for-validation). This keeps validation of each parameter organized into its own method.

Reply:
I imagine we will have to make a new validator for the yellow and grey letters?